bahmni_product/models/product_uom.py

```diff
@@ -3,22 +3,6 @@
 from odoo import models, fields, api
 
 
-class ProductUom(models.Model):
-    _inherit = 'uom.uom'
```

Comment:
Why are we removing this inheritance ? Models should not be defined by the initializer

Reply:
@mohan-13 This ProductUom model which inherits 'uom.uom' was created with the sole purpose of adding uuid field. uuid was. UUID is never updated or used anywhere in during sync/execution. I felt that it would be better to remove this duplicate method.